rust/json-gen/src/main.rs

```diff
@@ -109,7 +109,8 @@ fn main() {
     gen_json_schema!(AuxiliaryDataHash);
     gen_json_schema!(PoolMetadataHash);
     gen_json_schema!(VRFKeyHash);
-    gen_json_schema!(BlockHash);
```

Comment:
Just FYI, this was actually a type mistake in the old bindings. Block body hash and block header hash are different things (but coincidentally both happen to be 32 bytes so it didn't cause any issue)